sky/shell/android/org/domokit/sky/shell/SkyApplication.java

```diff
@@ -34,10 +34,11 @@
 public class SkyApplication extends BaseChromiumApplication {
     static final String SNAPSHOT = "snapshot_blob.bin";
     static final String APP_BUNDLE = "app.skyx";
+    static final String MANIFEST = "sky.yaml";
 
     private static final String TAG = "SkyApplication";
     private static final String PRIVATE_DATA_DIRECTORY_SUFFIX = "sky_shell";
-    private static final String[] SKY_RESOURCES = {"icudtl.dat", SNAPSHOT, APP_BUNDLE};
+    private static final String[] SKY_RESOURCES = {"icudtl.dat", SNAPSHOT, APP_BUNDLE, MANIFEST};
```

Comment:
Yeah I was thinking the version check would download the manifest, check versions, and replace manifests if it needs to do the update. I may rethink this part though.